sources/core/objects/Human.h

```diff
@@ -42,11 +44,22 @@ class Human: public IMob
 
     void UpdateOverlays();
 
-    int GetHealth() { return health_; }
+    auto Random() const -> double;
+
+    int GetHealth() { return CalculateHealth(); }
 
     void RotationAction(IdPtr<IOnMapBase> item);
     void PullAction(IdPtr<IOnMapBase> item);
     void StopPull();
+    void AddDamage(int brute, int suffocation, int burn);
+    
+    int GetBurnDamage() {return suffocation_damage_;}
+    int GetSuffocationDamage() {return burn_damage_;}
+    int GetBruteDamage() {return brute_damage_;}
+    bool IsDead() {return dead_;}
```

Comment:
In such cases it should be `int GetBruteDamage() { return brute_damage_; }` (spaces after `{` and before `}`).